File: Difficulty: Easy/Reverse Digits/reverse-digits.java
class Solution {
    public int reverseDigits(int n) {
        // Code here
        char[] s=Integer.toString(n).toCharArray();
        int r=0,l=s.length-1;
        while(r<l){
            char temp=s[r];
            s[r]=s[l];
            s[l]=temp;
            r++;
            l--;
        }
        return Integer.valueOf(new String(s));
    }
}

File: Difficulty: Easy/Reverse Digits/reverse-digits.cpp
class Solution {
  public:
    int reverseDigits(int n) {
        // Code here
        string s=to_string(n);
        int i=0;
        int l=s.size()-1;
        while(i<l){
            char temp=s[i];
            s[i]=s[l];
            s[l]=temp;
            i++;
            l--;    
            
        }
        return stoi(s);
    }
};